platform-coop-toolkit/coop-library-framework
# Add rights and custom rights fields (resolves #5)

## Description

Adds:

- Rights menu as per https://github.com/platform-coop-toolkit/learning-commons-framework/issues/5#issuecomment-535709682.
- Custom Rights field which can be enabled by selecting "Custom" from the Rights menu.

## Steps to test

1. Add a resource.
2. Select one of the options for Rights, save.
3. Change the rights menu to "Custom…".
4. Enter a Custom Rights statement, save.

**Expected behavior:** Rights are saved under various circumstances. Selecting or deselecting "Custom…" from the Rights menu enables/disables the Custom Rights field.

## Additional information

Not applicable.

## Related issues

- Resolves #5.
